pkgs/xdg-desktop-portal-cosmic/package.nix

```diff
@@ -45,14 +47,13 @@ rustPlatform.buildRustPackage rec {
   postPatch = ''
     substituteInPlace data/org.freedesktop.impl.portal.desktop.cosmic.service \
       --replace-fail 'Exec=/bin/false' 'Exec=${lib.getExe' coreutils "true"}'
+    substituteInPlace Makefile \
+      --replace-fail 'rev' '${lib.getExe' util-linux "rev"}'
   '';
 
   postInstall = ''
-    mkdir -p $out/share/{dbus-1/services,icons,xdg-desktop-portal/portals}
-    cp -r data/icons $out/share/icons/hicolor
-    cp data/*.service $out/share/dbus-1/services/
-    cp data/cosmic.portal $out/share/xdg-desktop-portal/portals/
-    cp data/cosmic-portals.conf $out/share/xdg-desktop-portal/
+    make prefix=$out CARGO_TARGET_DIR=target/${stdenv.hostPlatform.rust.cargoShortTarget} install
```

Comment:
This should be done with `dontCargoInstall` and using the stdenv `installPhase` for `make` instead. I'll make the change when I have time

Reply:
I can update it

Reply:
Actually yeah just look at the cosmic-comp derivation which already does this :)